src/test/util/strings_util/containing_substrings_test.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2020 Anastasiia Birillo, Elena Lyulina
+
+import pytest
```

Comment:
Pytest is not a standard library, so please, move this import into a separated second group